tasks/__init__.py

```diff
@@ -940,6 +963,7 @@ def generate_self_signed(domain):
     ).format(domain)
     run(cmd)
 
+
```

Comment:
FWIW, our flake config explicitly ignores the "2-newline" rule since it leads to unnecessary whitespace. This is fine though.